x-pack/plugins/security_solution/public/management/components/endpoint_responder/action_success.tsx

```diff
@@ -0,0 +1,37 @@
+/*
```

Comment:
This component is currently only being used for Kill/Suspend, however, it is generic, so it could be used with all Actions. Perhaps we can refactor existing commands for 8.5